lib/saluki-event/src/lib.rs

```diff
@@ -25,11 +25,11 @@ pub enum DataType {
     /// Metrics.
     Metric,
 
-    /// Logs.
-    Log,
+    /// Datadog Events.
+    EventD,
 
-    /// Traces.
-    Trace,
+    /// Service checks.
+    ServiceCheck,
```

Comment:
The old variants -- `Log` and `Trace` -- didn't actually map to any possible variant in `Event`, so I took this as a good chance to align it with `Event`.